piksi_tools/fileio.py

```diff
@@ -455,6 +457,13 @@ def get_args():
     return parser.parse_args()
 
 
+def raw_filename(str_filename):
+    """Return a filename in raw bytes from a command line option string."""
+    # Non-unicode characters/bytes in the command line options are decoded by
+    # using 'surrogateescape' and file system encoding, and this reverts that.
```

Comment:
Is there a reference for this?  Can we link it?

Reply:
I added two references:
* https://www.python.org/dev/peps/pep-0383/
* https://docs.python.org/3/library/os.html#file-names-command-line-arguments-and-environment-variables